### PR TITLE
Swapna_Assignment_34_Done_10+40_min_updated_as_per_changes

### PR DESCRIPTION
**Pull request by SwapnaWJun20**
image

image

### CODE REVIEW

File: src/swapnaWJun20/RemoveTechnoArrayList.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+public class RemoveTechnoArrayList {
+
+	void removeWord(ArrayList<String> input, String toBeRemoved) {
+		for (int index = 0; index < input.size(); index++) {
+			if (input.contains(toBeRemoved)) {
+				input.remove(toBeRemoved);
+			}
+		}
+		System.out.println("Output : " + input);
+	}
```
Comment (AnupSJun20) [inline src/swapnaWJun20/RemoveTechnoArrayList.java:14]: This is good but as we don't know the iteration, while is preferable.

File: src/swapnaWJun20/ReplaceTechnoArrayList.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+		input.add("Maulik");
+		input.add("Techno");
+		input.add("Krishna");
+		input.add("Techno");
+		input.add("Credits");
+		input.add("Techno");
+		System.out.println("Input : " + input);
+		String toBeRemoved = "Techno", toReplaceWith = "Credits";
+		new ReplaceTechnoArrayList().replaceWord(input, toBeRemoved, toReplaceWith);
+	}
```
Comment (AnupSJun20) [inline src/swapnaWJun20/ReplaceTechnoArrayList.java:35]: This looks good.

**Review by AnupSJun20 — APPROVED** [review]
Code looks good. So approving this.

File: src/swapnaWJun20/RemoveTechnoArrayList.java (hunk 1 above)
Comment (SwapnaWJun20) [reply src/swapnaWJun20/RemoveTechnoArrayList.java:14]: Thanks Anup. Code has updated as per the comment.

**Review by KrishnaKTechnocredits — APPROVED** [review]
Good job dear.

**Pull request merged**